internal/vrf/create.go

```diff
@@ -0,0 +1,77 @@
+package vrf
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"strings"
+
+	metal "github.com/equinix-labs/metal-go/metal/v1"
```

Comment:
goimports format sequence we can follow - 1st block go runtime imports, 2nd block 3rd party imports, 3rd block local packages import

Reply:
In other words: `goimports -l "github.com/equinix-labs/metal-go" -w`